.github/workflows/build.yaml

```diff
@@ -68,8 +68,9 @@ jobs:
     secrets: inherit
     uses: rapidsai/shared-workflows/.github/workflows/wheels-build.yaml@cuda-12.9.0
     with:
-      # This selects "ARCH=amd64 + the latest supported Python + CUDA".
-      matrix_filter: map(select(.ARCH == "amd64")) | group_by(.CUDA_VER|split(".")|map(tonumber)|.[0]) | map(max_by([(.PY_VER|split(".")|map(tonumber)), (.CUDA_VER|split(".")|map(tonumber))]))
+      # This selects "ARCH=amd64 + Python 3.12 + CUDA".
```

Comment:
This would drop Python 3.10 and 3.11, which I don’t think we want? Just say “not 3.13” here.

Reply:
Same in pr.yaml.

Reply:
It was already dropping Python 3.10 and 3.11 with the `max_by(.PY_VER)` -- I'm just telling it to use 3.12 instead of the new max of 3.13

Reply:
Ahhhh I missed that this is a pure wheel. (See `pure-wheel: true` below). That's fine, then.